torch_xla/experimental/pjrt.py

```diff
@@ -7,9 +7,9 @@
 
 aliases = [
     runtime.addressable_device_count,
-    runtime.device_attributes,
```

Comment:
You'll have to rename these two functions like `rendezvous` below, otherwise the API will just be `pjrt.physical_device_attributes`. e.g. `device_attributes = deprecated(this_module, runtime.physical_device_attributes)`

Reply:
done